tests/models/bert_japanese/test_tokenization_bert_japanese.py

```diff
@@ -88,14 +92,14 @@ def test_maximum_encoding_length_single_input(self):
         pass  # TODO add if relevant
 
     def test_full_tokenizer(self):
-        tokenizer = self.tokenizer_class(self.vocab_file)
+        tokenizer = self.tokenizer_class(vocab_file=self.vocab_file, spm_file=None)
```

Comment:
That change should no be necessary once the breaking change is fixed.

(Same four times below)